packages/amplify-provider-awscloudformation/src/configuration-manager.js

```diff
@@ -645,6 +742,19 @@ async function getAwsConfig(context) {
         region: awsConfigInfo.config.region,
       };
     }
+  } else if (awsConfigInfo.configLevel === 'amplifyAdmin') {
+    try {
+      const meta = stateManager.getMeta();
+      const appId = _.get(meta, ['providers', 'awscloudformation', 'AmplifyAppId']);
+      const authConfig = await getRefreshedTokens(appId, context.print);
+      const { idToken, IdentityId, region } = authConfig;
+      awsConfig = {
+        ...(await getAdminCredentials(idToken, IdentityId, region)).Credentials,
+        region,
+      };
+    } catch (e) {
+      throw new Error('Failed to get Amplify Admin credentials.');
```

Comment:
We don't include the original error, we should. Or remove the whole try catch block and let it bubble up it has no real value this way.